### DISCUSSION TITLE
is Tpot affected by the ongoing log4j issue ???

### DISCUSSION THREAD

**Discussion by arnux72** [telekom-security/tpotce#964] [General]
Hi all,

refering to CVE-2021-44228 - (RCE 0-day exploit found in log4j) elasticsearch might be infected as well. Do you have an idea about this and this vulnerability is ralated to tpot as well?

Thanks for your kind assistance.

**Comment by arnux72** [comment]
Now seen:

t3chn0m4g3 remove log4j JndiLookup Class         0ef2e89      10 minutes ago

Perfect!! Thanks

**Comment by t3chn0m4g3** [comment]
@arnux72 Please see #965.

**Comment by arnux72** [comment]
@t3chn0m4g3 many thanks. Great job. All is working fine.
Thanks a lot for your effort.

**Reply by t3chn0m4g3** [reply]
Thank you! Great to hear 😃
